test/jdk/java/util/Properties/StoreReproducibilityTest.java

```diff
@@ -137,7 +137,7 @@ private static void testWithSecMgrExplicitPermission() throws Exception {
             final ProcessBuilder processBuilder = ProcessTools.createJavaProcessBuilder(
```

Comment:
ProcessTools.createJavaProcessBuilder came up in another PR because it doesn't prepend the VM and java opts. As all usages of PB are being updated in this test then it makes me wonder if it should be changed to use createTestJvm while you're there.

Reply:
Actually, I was limiting the change to a single JVM invocation 😄
But yes, it is appropriate to replace the invocation. Replaced them all.

Reply:
> But yes, it is appropriate to replace the invocation. Replaced them all.

Thanks